slice.go

```diff
@@ -0,0 +1,119 @@
+package golds
+
+// Slice is a sequence of values.
+// Basically it's an utility wrapper around a plain go slice.
+type slice[E any] []E
+
+// NewSlice puts provided values into resulting slice.
+//export
+func newSlice[E any] (vv ... E) slice[E] {
+	return slice[E](vv)
+}
+
+// Repeat returns a new Slice[E] with n copies of v.
+func repeat[E any](n int, v E) slice[E] {
+	switch{
+	case n == 0:
+		return nil
+	case n < 0:
+		panic("golds.Repeat: negative Repeat count")
+	}
+	var s = make(slice[E], n)
```

Comment:
can be in default-case

Reply:
I want to provide meaningful panic message for n < 0 cases